index.html

```diff
@@ -29,30 +29,30 @@ <h2>
       <div class="features">
         <div class="feature">
           <span class="fa fa-chart-line fa"></span>
-          <h5>Analyze event streams</h5>
+          <h5>Power applications where low latency really matters</h5>
           <p>
-            Druid provides fast analytical queries, at high concurrency, on event-driven data. Druid can instantaneously ingest streaming data and provide sub-second queries to power interactive UIs.
+            Druid is a real-time analytics database that excels at providing immediate data visibility and understanding. Easily load streaming or batch data, and answer ad-hoc analytic queries on billions of rows in under a second.
           </p>
         </div>
         <div class="feature">
-          <span class="fa fa-lightbulb fa"></span>
-          <h5>Utilize reimagined architecture</h5>
+          <span class="fa fa-forward fa"></span>
+          <h5>Build event-driven data stacks</h5>
           <p>
-            Druid is a new type of database that combines ideas from <a href='https://en.wikipedia.org/wiki/Online_analytical_processing'>OLAP/analytic databases</a>, <a href='https://en.wikipedia.org/wiki/Time_series_database'>timeseries databases</a>, and <a href='https://en.wikipedia.org/wiki/Full-text_search'>search systems</a> to enable new use cases in streaming architectures.
+            Druid is a query layer that that natively integrates with message buses (<a href='http://kafka.apache.org/'>Kafka</a>, <a href='https://aws.amazon.com/kinesis/'>AWS Kinesis</a>, etc) and data lakes (<a href='https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html'>HDFS</a>, <a href='https://aws.amazon.com/s3/'>AWS S3</a>, etc). Build end-to-end data stacks ideal for streaming data.
```

Comment:
I usually describe Druid as _not_ a query layer, in the sense that it stores data inside itself, and doesn't query it live from somewhere else.